### PR TITLE
Game logic of tic tac toe 

### PR DESCRIPTION
**Pull request by jaspreet-singh-sahota**
## Just for fun we have changed the tokens "X" and "O" to "rum" and "vodka"
- This is a modern Tic-Tac-Toe game.
- Try running the game with wrong arguments first, and have fun.
- Waiting for your feedback.

### CODE REVIEW

File: lib/position.rb
Comment (MauricioRobayo) [inline lib/position.rb]: Use `attr_reader` instead of `attr_accessor` is you are only going to read, not write, the variable. Check your use of `attr_accessor`.

File: lib/position.rb
Comment (MauricioRobayo) [inline lib/position.rb]: No `puts/print` and `get.chomp` outside `main.rb` file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

**Review by MauricioRobayo — CHANGES_REQUESTED** [review]
Very nice project, I had a lot of fun playing with it, well done!

We need to fix some issues before continuing:

* [x] Do not make any changes in [config files](https://github.com/microverseinc/linters-config/tree/master/ruby) - they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/))
* [x] Please fix Stickler/Rubocop issues.
* [x] No `puts/print` and `get.chomp` outside `main.rb` file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))
* [x] Use attr_readers/attr_writers instead of attr_accessors when possible to avoid writing that property from outside the class  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/attr_accessor.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/)).
* [x] Avoid global variables, it breaks the OOP encapsulation principle: https://www.thoughtco.com/global-variables-2908384.

Status: __Request changes__.

---

Optional:

* The board looks a little misaligned which makes it difficult to follow:

image

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project requires changes.

**Comment by pelzolga123** [comment]
## Hi Team!  
### Your project looks amazing, good job so far :+1: 
Still, you need to make some more changes before this project can be approved:

* [x] You removed board class after first review, that's not very good. The project should use at least two classes. The code of the project should be logically divided between classes. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/classes.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))
* [x] All linter errors should be fixed. 
* [x] Stickler should not ignore `bin` folder. As previous review asked you should not make any changes in config files .

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Comment by rammazzoti2000** [comment]
Hello @pelzolga123, thank you for the feedback, very helpful.
Completed all the required changes, kindly waiting for your feedback, thank you.

**Review by MauricioRobayo — APPROVED** [review]
This is it, you got it!

Congratulations, please move on.

Status: __Approved__.

**Pull request merged**
